diff/walking/differ.go

```diff
@@ -55,6 +56,26 @@ func NewWalkingDiff(store content.Store) diff.Comparer {
 	}
 }
 
+func readonlyOverlay(opt []string) []string {
```

Comment:
Suggest to add comments or rename this function to something like `removeUpperdirForReadonlyOverlay`, to be more descriptive.